go.mod

```diff
@@ -6,7 +6,7 @@ replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
 	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-	google.golang.org/grpc => google.golang.org/grpc v1.33.2
+	//google.golang.org/grpc => google.golang.org/grpc v1.33.2
```

Comment:
@alexstratos it seems this replacement has to be commented out for the ipfs kubo lib to work. I'll discuss with @jialbai about this line

Reply:
confirmed with @jialbai , this line can be removed as long as the app can be compiled